src/cxy_ctl.erl

```diff
@@ -58,14 +61,43 @@ make_process_dictionary_default_value(Key, Value) ->
 
 %% Use a raw tuple because two record structs with the same key would
 %% be needed, or redundant record name in key position.
+
+%% Cxy counters tuple...
 -define(MAX_PROCS_POS,    2).
 -define(ACTIVE_PROCS_POS, 3).
 -define(MAX_HISTORY_POS,  4).
 
-make_proc_values(Task_Type, Max_Procs_Allowed, Max_History) ->
+%% Cumulative performance moving average tuple...
+-define(CMA_TYPE_POS,        1).
+-define(CMA_SPAWN_CMA_POS,   2).
+-define(CMA_EXEC_CMA_POS,    3).
+-define(CMA_RING_SIZE_POS,   4).
+-define(CMA_SLOW_FACTOR_POS, 5).
+
+%% Read all values using update_counter to avoid releasing write_lock for a read_lock.
+-define(CMA_READ_CMD,
+        [{?CMA_SPAWN_CMA_POS,   0},
+         {?CMA_EXEC_CMA_POS,    0},
+         {?CMA_RING_SIZE_POS,   0},
+         {?CMA_SLOW_FACTOR_POS, 0}
+        ]).
```

Comment:
This is a buffer intended to hold 1M + entries. And an ets table intended to possibly hold multiple buffers. Making a record adds a record name to the tuple, but no information. It is clearly wasted space and not premature.